gammapy/catalog/hess.py

```diff
@@ -512,41 +514,87 @@ def spatial_model(self):
             model.parameters[name].error = value
         return model
 
-    def sky_model(self, which="best"):
+    def sky_model(self, which="best", components_status="independent"):
```

Comment:
Here I'm not a 100% whether to expose the options in `.sky_model()`. Ideally we could achieve uniformity in the returned types. I could imagine e.g. to introduce `SourceCatalogObjectHGPS.to_models(link_components=)`  or `components_models()` as well. `.sky_model()`then reduces to `.to_models().to_template_spatial_model()` an always returns a `SkyModel`